zeppelin-interpreter/src/main/java/org/apache/zeppelin/cluster/ClusterManagerServer.java

```diff
@@ -310,18 +311,51 @@ public void broadcastClusterEvent(String msg) {
     }
   }
 
-  private BiFunction<Address, byte[], byte[]> subscribeClusterEvent = (address, data) -> {
+  private BiFunction<Address, byte[], byte[]> subscribeClusterIntpEvent = (address, data) -> {
+    String message = new String(data);
+    if (LOGGER.isDebugEnabled()) {
```

Comment:
why is `if (LOGGER.isDebugEnabled()) {` needed? `LOGGER.debug` should be enough?

Reply:
Because need to synchronize the `create`, `rename`, `delete`, set `read permissions`, `write permissions`, `execute permissions`, so need to synchronize the more information, there will be a lot of synchronization information in the log file.
So need add `if (LOGGER.isDebugEnabled())`

Reply:
I'm not sure I understand. you are saying logging is useful for debugging, and there could be a lot of information.
isn't debug log will be filtered out if debug log is not enabled?

Reply:
In the development phase, the debug information is very useful for developers, but after the development is completed, the debug information will be very much. In the log log file, there is a lot of synchronization information, which affects the system administrator to check whether the system is normal.
So, I think these debug messages don't need to be printed to the log file after development. :-)